cluster/addons/cluster-loadbalancing/glbc/glbc.yaml

```diff
@@ -1,3 +1,8 @@
+{% set kube_uid = "" -%}
```

Comment:
Thanks for heads-up. I am waiting for this PR to be merged, then I can test my change to gci support